example/example/settings.py

```diff
@@ -153,14 +153,14 @@
             'BACKEND': 'django.template.backends.django.DjangoTemplates',
             'OPTIONS': {
                 'context_processors': [
-                    'django.contrib.auth.context_processors.auth',
-                    'django.core.context_processors.debug',
-                    'django.core.context_processors.i18n',
-                    'django.core.context_processors.media',
-                    'django.core.context_processors.request',
-                    'django.core.context_processors.tz',
-                    'django.core.context_processors.static',
-                    'django.contrib.messages.context_processors.messages',
+                'django.contrib.auth.context_processors.auth',
```

Comment:
I haven't seen the flake8 results, but I think these will need to stay indented.